test/Polly.Core.Tests/Retry/RetryResilienceStrategyTests.cs

```diff
@@ -89,9 +89,9 @@ public void ExecuteAsync_MultipleRetries_EnsureDiscardedResultsDisposed()
         // assert
         result.IsDisposed.Should().BeFalse();
         results.Count.Should().Be(_options.MaxRetryAttempts + 1);
-        results.Last().IsDisposed.Should().BeFalse();
+        results[results.Count - 1].IsDisposed.Should().BeFalse();
 
-        results.Remove(results.Last());
+        results.Remove(results[results.Count - 1]);
```

Comment:
Does using the terser `[^1]` work here (and in the other files) instead?

Reply:
Unfortunately `[^1]` can not be used due net481